tests/test_osut.py

```diff
@@ -5001,6 +5001,15 @@ def test34_generated_skylight_wells(self):
         ratio = sky_area1 / rm2
         self.assertAlmostEqual(round(ratio, 2), srr)
 
+        # Reset attic default construction set for insulated interzone walls.
+        opts = dict(type="partition", uo=0.3)
+        construction = osut.genConstruction(model, opts)
+        self.assertAlmostEqual(osut.rsi(construction, 0.150), 1/0.3, places=2)
+        self.assertTrue(ia_set.setWallConstruction(construction))
+        if o.logs(): print(o.logs())
+
+        model.save("./tests/files/osms/out/office_attic.osm", True)
```

Comment:
Both North & South roof surfaces now inherit leader lines and cutouts to accommodate skylight wells. Although roof surfaces inherit such extended vertices (consult generated _office_attic.osm_ file with a text editor), the OpenStudio Application won't necessarily render them - that's OK.

<img width="975" height="679" alt="office_wells" src="https://github.com/user-attachments/assets/6ecd448d-1578-465b-8c46-92db8411a61e" />


There's now a single _eplusout.err_ file warning on collinear vertices - which is expected. Otherwise, slight (expected) changes in results.

<img width="582" height="210" alt="skyoff2_res" src="https://github.com/user-attachments/assets/e82460d9-da31-4ec2-bfa2-566ffbd58bee" />

All good for now. More testing to come.